react/TaxTooltip.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FunctionComponent } from 'react'
+import { Tooltip, IconInfo } from 'vtex.styleguide'
+import TranslateTotalizer from 'vtex.totalizer-translator/TranslateTotalizer'
+
+import FormattedPrice from './components/FormattedPrice'
+
+const TaxTooltip: FunctionComponent<Props> = ({ taxesTotals }) => {
```

Comment:
Does it make sense?

```suggestion
const TaxInfo: FunctionComponent<Props> = ({ taxesTotals }) => {
```

Reply:
It does, but does it doesn't make sense? 🤔

Reply:
I like it, thanks

Reply:
What I thought when I saw the `TaxTooltip` name was that its API would be similar to the Styleguide's `Tooltip` and would be called with some children.

Reply:
Done